### ISSUE TITLE
Fall cleaning

### ISSUE THREAD

**Issue by pochedls** [pochedls/xagg#9]
We have various versions of xml libraries lying around. 

It's time to wipe them out to avoid confusing people: 

`rm -rf ...`

* /work/cmip-dyn/
* /work/cmip5-dyn/
* /work/cmip5-test/
* /work/cmip5/

**Comment by pochedls** [comment]
@durack1 - let me know your thoughts on this one. All of my dyn / test directories can go.

**Comment by durack1** [comment]
@pochedls I agree, pinging @gleckler1 @lee1043 @bonfils2 @mzelinka for signoff

**Comment by lee1043** [comment]
@pochedls agreed, thanks for maintaining this!

**Comment by bonfils2** [comment]
@pochedls same here, thank you!

**Comment by mzelinka** [comment]
RIP
